.travis.yml

```diff
@@ -1,37 +1,18 @@
 language: ruby
+cache: bundler
 rvm:
-  - 2.1.10
-  - 2.2.1
-  - 2.2.2
-  - 2.2.10
-  - 2.3.8
-  - 2.4.9
   - 2.5.7
   - 2.6.5
   - 2.7.0
-before_install:
-  - gem install bundler -v '~> 1.0'
+  - jruby-head
```

Comment:
Is there any reason why we want to test the jruby-head here? Should we test against jruby 9.2?

Reply:
[It seems that JRuby 9.2 is working on Travis.](https://travis-ci.org/github/jeremytregunna/ruby-trello/jobs/681528458)